### ISSUE TITLE
Setup App: IDEA / Eclipse project files can't be generated

### ISSUE THREAD

**Issue by cluder** [libgdx/libgdx#5781]
#### Issue details
When using the Setup App, trying to create IDEA or Eclipse project files fails with the following errors:
`FAILURE: Build failed with an exception.
 What went wrong:
Task 'eclipse' not found in root project 'libgdxtest1'.`

`FAILURE: Build failed with an exception.
What went wrong:
Task 'idea' not found in root project 'libgdxtest1'.
`

#### Reproduction steps/code
- download current Setup App https://libgdx.badlogicgames.com/nightlies/dist/gdx-setup.jar
- click Advanced
- enable Eclipse / IDEA project file checkboxes
- click generate

#### Version of LibGDX and/or relevant dependencies
Current libGDX version as of 30.08.2019

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [x ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by Awes0meM4n** [comment]
Same for me, but finally I try to not check Eclipse option. Then I imported the gradle project and the integration with Eclipse works fine.
I only imported, run check task in main project and Refresh Gradle Project from Eclipse.
Try it!

**Comment by Tom-Ski** [comment]
Yes, thats the recommended way. I removed the other plugins as they were causing confusion but forgot to remove these settings.

**Comment by bobsort** [comment]
There is a tiny bit of issue with setup app, OpenJDK doesn't run it without knowing which class to run!

**Comment by Tom-Ski** [comment]
Which version?

**Comment by bobsort** [comment]
I believe all versions, mine is jdk-12.0.1, It seems OpenJDK doesn't look for main method in classes and you need to call it from java.exe params

**Comment by payne911** [comment]
I would like to add that checking the "IDEA" advanced option results in an output message that goes like this:

```Generating app in C:\...
Executing 'C:\.../gradlew.bat clean --no-daemon idea'
Downloading https://services.gradle.org/distributions/gradle-5.4.1-bin.zip
...................................................................................
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/5.4.1/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing

FAILURE: Build failed with an exception.

* What went wrong:
Task 'idea' not found in root project 'PieMenu'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 32s
Done!
To import in Eclipse: File -> Import -> General -> Existing Projects into Workspace
To import to Intellij IDEA: File -> Open -> YourProject.ipr```

Why would I be suggested to open the project in Eclipse when I selected "IDEA" specifically?
Also, there was no `YourProject.ipr` file generated.
Moreover, using the generated `build.gradle` file to open as a Project in IDEA gives an error saying "Gradle project sync failed.". It also automatically included libraries I did not check in the setup jar, namely `Ashely`, `Ai` and `Box2d`.

**Comment by MaanooAk** [comment]
@Tom-Ski Could we put back the eclipse project name defined on build.gradle. Now all the imported project are called `core` instead of `projectName-core`

**Comment by mgsx-dev** [comment]
for reference, i suggested it as well in original PR, see : https://github.com/libgdx/libgdx/pull/5700#issuecomment-513907532

**Comment by MrStahlfelge** [comment]
Is this still a thing or is Eclipse setup working again?

**Comment by mgsx-dev** [comment]
it has been fixed in #5890, i'm closing.
